bindings/pydeck-carto/pydeck_carto/sources.py

```diff
@@ -0,0 +1,49 @@
+import pydeck as pdk
+
+
+def base_source(connection_name: str, access_token: str, api_base_url: str):
+    return {
+        "connectionName": connection_name,
+        "accessToken": access_token,
+        "apiBaseUrl": api_base_url,
```

Comment:
Just thinking, because we lose the types here, if it would be worth adding some run time checks to enforce the required params are passed, e.g. `accessToken`

Reply:
Done. ✅